### ISSUE TITLE
Design request: redesign dapps page

### ISSUE THREAD

**Issue by konopkja** [ethereum/ethereum-org-website#13679]
This is a design request for anyone to redesign our https://ethereum.org/en/dapps . Ideally, take a holistic approach, remove excessive info and freshen up the UI.

It has been a long time since this page has been updated and the ecosystem has evolved since.

- The mobile design is not well optimized currently hindering the mobile UX
- The content is too large in its scope for one page (the focus should be on doing, less learning)
- Must follow design system - https://www.figma.com/community/file/1134414495420383395

We are looking for fresh takes and ideas for this page, this issue is a call for design exploration and ideation in figma

**Comment by corwintines** [comment]
I would recommend we use our Product List design (featured on find wallet page, and soon networks page) as a product table here

**Comment by Rushhaabhhh** [comment]
Hello @konopkja , I’d like to address this issue also can you please elaborate the changes needed in the UI. 
 My work is available here : https://react-portfolio-lime-five.vercel.app/

**Comment by konopkja** [comment]
> https://react-portfolio-lime-five.vercel.app/

Hi @Rushhaabhhh this is a designer request rather than just dev work. Its an open request to ideally reimagine the page and provide fresh ideas and propose changes yourself with some rationale why its a good change.

**Comment by yaasiinaxmed** [comment]
Hi @konopkja, I would like to take on this issue. Could you assign it to me?

**Comment by konopkja** [comment]
> Hi @konopkja, I would like to take on this issue. Could you assign it to me?

hi, happy to do so! @yaasiinaxmed  did you read previous comments in this issue to understand that this isnt just dev work?

**Comment by yaasiinaxmed** [comment]
> > Hi @konopkja, I would like to take on this issue. Could you assign it to me?
> 
> hi, happy to do so! @yaasiinaxmed did you read previous comments in this issue to understand that this isnt just dev work?

@konopkja, just to clarify, I understand that the task involves creating a fresh UI in Figma first before making any code changes, right?

**Comment by konopkja** [comment]
> > > Hi @konopkja, I would like to take on this issue. Could you assign it to me?
> > 
> > 
> > hi, happy to do so! @yaasiinaxmed did you read previous comments in this issue to understand that this isnt just dev work?
> 
> @konopkja, just to clarify, I understand that the task involves creating a fresh UI in Figma first before making any code changes, right?

yes, ideally mostly reusing components we already have, such as https://ethereum.org/en/wallets/find-wallet/ or other, you should follow design system linked above.

Once you share figma i and the team can give you continuous feedback so that we ensure some quality and consensus.

**Comment by yaasiinaxmed** [comment]
> > > > Hi @konopkja, I would like to take on this issue. Could you assign it to me?
> > > 
> > > 
> > > hi, happy to do so! @yaasiinaxmed did you read previous comments in this issue to understand that this isnt just dev work?
> > 
> > 
> > @konopkja, just to clarify, I understand that the task involves creating a fresh UI in Figma first before making any code changes, right?
> 
> yes, ideally mostly reusing components we already have, such as https://ethereum.org/en/wallets/find-wallet/ or other, you should follow design system linked above.
> 
> Once you share figma i and the team can give you continuous feedback so that we ensure some quality and consensus.

Thanks for the details, @konopkja! I’ll start by creating the new design in Figma, using components from the Find Wallet page and following the design system. Once I have a draft, I’ll share it for feedback from you and the team.

**Comment by yaasiinaxmed** [comment]
**Hi @konopkja,**

I finished my redesign of the Dapps page! I made some smaller updates instead of a full redesign:

- Removed the category details to keep things simple
- Updated the Dapps list layout to be easier to read
- Changed the style of category buttons
- Replaced the hero section image
- Made a few small tweaks to other sections for a cleaner look

Here’s the link to the Figma file: [Figma Redesign - Dapps Page](https://www.figma.com/design/5hBdpCf1HYo9oPm7rK3QxO/Dapps-Page-Redesign---Yasin-Ahmed---Oct-2024?node-id=0-1&t=Y168zmYVVvjGjTWn-1)

Please let me know if there’s anything else you’d like me to change!

Thanks!

**Comment by konopkja** [comment]
hi provided some feedback in the figma file, i think we need to readjust the focus of redesign a little - let me know your thoughts!

**Comment by yaasiinaxmed** [comment]
Hi @konopkja, I've made some updates in the Figma file based on the feedback. Please review it when you can and let me know if it meets your expectations.

**Comment by wackerow** [comment]
Hey folks, circling through... @yaasiinaxmed appreciate the work put in! This initiative is actually full force right now with multiple folks from the foundation onboard helping out. The canonical PR for the apps updates are now in #15623 which @corwintines is leading. Re-assigned to him and have linked that PR to close this issue when merged.

**Comment by konopkja** [comment]
Hello, closing this as the internal team completely revamped this page and will soon ship it.
